### PR TITLE
Update 2015-01-25-dynamic-casting-in-swift.md

### PR DESCRIPTION
**Pull request by saniul**
Provide an example for the `id` function

### CODE REVIEW

File: source/2015-01-25-dynamic-casting-in-swift.md [hunk 1]
```diff
@@ -43,6 +43,21 @@ Now, we can just write:
...
+For example, the `id` function can be used when passing the extracted value into a typed function:
+```swift
+func doubleMaybe(i: Int?) -> Int? {
+    if let i = i {
+        return i * 2
+    }
+    return nil
+}
+
+//Note that we don't have to explicitly cast to Int?
```
Comment (segiddins) [inline source/2015-01-25-dynamic-casting-in-swift.md]: space after `//`?

**Comment by segiddins** [comment]
:+1: with that one change!

File: source/2015-01-25-dynamic-casting-in-swift.md (hunk 1 above)
Comment (saniul) [inline source/2015-01-25-dynamic-casting-in-swift.md]: Done

**Comment by saniul** [comment]
:boom:

**Pull request merged**